src/components/Footer/FooterInf.jsx

```diff
@@ -1,63 +1,195 @@
-                <div className='footer-main-2'>
-                <ul>FLOWBASE
-                    <li>More Cloneables</li>
-                  </ul>
-                </div>
-            </div>
-            <hr className="line"></hr>
-            <div className="maincopyright">
-            <div className='footer-copyright'>
-              <p>Built by&nbsp;</p>
-              <p className='copyrighttext'>Flowbase&nbsp;</p>
-              <p>Powered by&nbsp;</p>
-              <p className='copyrighttext'>Webflow</p>
-            </div>
-            <div className='logocopyright'>
-              <img src={logocopy1} alt="Logo1" id="logocopy1"/>
-              <img src={logocopy2} alt="Logo2" id="logocopy2"/>
-              <img src={logocopy3} alt="Logo3" id="logocopy3"/>
-            </div>
-            </div>
-        </div>
-      </footer>
+const FooterMain = styled.div`
+  width: 1200px;
+  height: auto;
+  padding-top: 99.8px;
+  padding-left: 120px;
+`;
+
+const FooterMain1 = styled.div`
+  height: 232px;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+`;
+
+const FooterInf1 = styled.div`
+  width: 584px;
+  height: auto;
+`;
+
+const LogoStyleFooter = styled.img`
+  width: 40px;
+  height: 51px;
+  padding-top: 0.2px;
+`;
+
+const TextFlogo1 = styled.p`
+  width: 243px;
+  height: 27px;
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  line-height: 27px;
+  letter-spacing: 0.36px;
+  padding-top: 25px;
+  margin: 0;
+  color: #546285;
+`;
+
+const TextFlogo2 = styled.p`
+  width: 243px;
+  height: 27px;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  line-height: 27px;
+  letter-spacing: 0.36px;
+  margin: 0;
+  color: #546285;
+`;
+
+const FooterMain2 = styled.div`
+  width: 189.3px;
+`;
+
+const FooterList = styled.ul`
+  font-family: 'Inter', sans-serif;
+  font-size: 13px;
+  line-height: 20px;
+  letter-spacing: 1.5px;
+  margin: 0;
+  list-style: none;
+  text-transform: uppercase;
+  color: #08090a;
+`;
+
+const FooterListItem = styled.li`
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  line-height: 20px;
+  letter-spacing: 0%;
+  padding-top: 28px;
+  text-transform: none;
+  white-space: nowrap;
+  color: #546285;
+`;
+
+const FooterLink = styled.a`
+  text-decoration: none;
+  color: inherit;
+`;
 
+const Line = styled.hr`
+  height: 1px;
+  margin-top: 99px;
+  margin-bottom: 0px;
+  padding: 0px;
+  background-color: #7e828f17;
+  opacity: 9%;
+`;
+
+const MainCopyright = styled.div`
+  display: flex;
+  padding-top: 71px;
+`;
+
+const FooterCopyright = styled.div`
+  display: flex;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  line-height: 20px;
+  letter-spacing: 0%;
+  color: #353448;
+`;
+
+const CopyrightText = styled.p`
+  color: #35b8be;
+`;
+
+const LogoCopyright = styled.div`
+  padding-left: 717px;
+`;
+
+const LogoCopy1 = styled.img`
+  padding-left: 12px;
+`;
+
+const LogoCopy2 = styled.img`
+  padding-left: 24px;
+`;
+
+const LogoCopy3 = styled.img`
```

Comment:
Component became very big with all that definition, could be moved to separate file for better readability